src/less/breadcrumb/ds6/breadcrumb-base.less

```diff
@@ -0,0 +1,35 @@
+.breadcrumb {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: flex-start;
+    list-style: none;
+    margin: 0;
+    li {
+        display: inline-block;
+        a {
+            color: #000000;
+            font-size: @ds6-font-size-12;
+            text-decoration: none;
+            &:hover {
+                color: @ds6-color-p002-blue;
```

Comment:
This should also be: `color: @ds6-color-link-default;` ... to remain consistent with elsewhere.

Reply:
This is what spec says. you could see it in sketch

Reply:
Hmmm. I haven't seen direct black as part of the standard. Can we confirm with the designer(s)?

Reply:
If you look at the variable, this is basically a pass through for the same color. 😉